### DISCUSSION TITLE
在 Chrome 下首页显示HTML代码，需要强制刷新才能显示网页。

### DISCUSSION THREAD

**Discussion by phpiscute** [vuepress/core#1289] [Q&A]
在 Chrome 下首页显示代码，需要强制刷新才能显示网页。

```html
<!DOCTYPE html>
<html lang="zh-CN">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <meta name="generator" content="VuePress 2.0.0-beta.26">
```

查看网络请求时，发现请求来源于 service worker，`content-type` 为 `text/html, application/json; charset=utf-8`，正常的 `content-type` 应该为 `text/html`。

<img width="607" alt="image" src="https://user-images.githubusercontent.com/87626576/227747648-cbfaa279-6371-4489-9dac-c83ecb622969.png">

**Comment by Mister-Hope** [comment]
Reproduction is needed for these kind of things.
